pkg/docker/entrypoint.sh

```
@@ -257,25 +257,31 @@ if [ -z "${PGADMIN_DISABLE_POSTFIX}" ]; then
 else
-    exec $SU_EXEC /venv/bin/gunicorn --limit-request-line "${GUNICORN_LIMIT_REQUEST_LINE:-8190}" --limit-request-fields "${GUNICORN_LIMIT_REQUEST_FIELDS:-100}" --limit-request-field_size "${GUNICORN_LIMIT_REQUEST_FIELD_SIZE:-8190}" --timeout "${TIMEOUT}" --bind "${BIND_ADDRESS}" -w 1 --threads "${GUNICORN_THREADS:-25}" --access-logfile "${GUNICORN_ACCESS_LOGFILE:--}" -c gunicorn_config.py run_pgadmin:app
+    ACCESS_LOG_ARGS="--no-access-log"
 fi
+
+TLS_ARGS=""
+if [ -n "${PGADMIN_ENABLE_TLS}" ]; then
+    TLS_ARGS="--ssl-keyfile /certs/server.key --ssl-certificate /certs/server.cert"
+fi
+
+# Keep the existing environment variables for backward compatibility.
+exec /venv/bin/granian --interface wsgi --workers 1 --blocking-threads "${GUNICORN_THREADS:-25}" ${ACCESS_LOG_ARGS} ${TLS_ARGS} ${BIND_ARGS} run_pgadmin:app
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)